aio_pika/queue.py

```diff
@@ -403,7 +403,12 @@ async def close(self, *_: Any) -> Any:
             while True:
                 msg = self._queue.get_nowait()
         except asyncio.QueueEmpty:
-            if msg is not None and not self._amqp_queue.channel.is_closed:
+            nack_ok = (
```

Comment:
I would like to see a log.warning that these messages have been received, not processed and will be lost.